src/js/components/dashboard/DashboardCoefficient.js

```diff
@@ -0,0 +1,56 @@
+import {
+    WORLD_POPULATION,
+    ONE_HUNDRED_PEOPLE
+} from '../../constants/constants';
+import create from '../../utils/create';
+import Dashboard from './Dashboard';
+
+export default class DashboardWithCoefficient extends Dashboard {
+    showGlobalData(infoParameter) {
+        const globalCases = document.querySelector('.global-cases__volume');
+        const totalParameterCoefficient = ((this.globalInfo[infoParameter] / WORLD_POPULATION)
+                                                                             * ONE_HUNDRED_PEOPLE).toFixed(2);
+        globalCases.textContent = totalParameterCoefficient;
+    }
+
+    showCountiesData(countryInfoParameter) {
+        const table = document.querySelector('.countries-cases__content');
+        table.innerHTML = '';
+        this.countriesInfo.forEach((country) => {
+            this.createCountryInfo(country, countryInfoParameter, table);
+        });
+        const lines = document.querySelectorAll('tr');
+        const columnNumber = 1;
+        const sortedRows = Array.from(lines)
+            .slice(1)
+            .sort((rowA, rowB) => Number(rowB.cells[columnNumber].innerHTML) - Number(rowA.cells[columnNumber].innerHTML));
```

Comment:
I recommend creating a separate function for this sort. You can place it in the utils folder